src/core/or/trace_probes_circuit.h

```diff
@@ -7,320 +7,16 @@
-  TP_FIELDS(
-    ctf_integer(uint32_t, circ_id, circ->global_identifier)
-    ctf_enum(tor_circuit, purpose, int, purpose, TO_CIRCUIT(circ)->purpose)
-    ctf_enum(tor_circuit, state, int, state, TO_CIRCUIT(circ)->state)
-    ctf_array_hex(char, fingerprint, hop->extend_info->identity_digest,
-                  DIGEST_LEN)
-  )
-)
-
-/*
- * OR circuit events.
- *
- * Tracepoint use the or_circuit_t object.
- */
-
-/* Tracepoint emitted when a new or circuit has been created. */
-TRACEPOINT_EVENT_INSTANCE(tor_circuit, or_circuit_t_class, new_or,
-  TP_ARGS(const or_circuit_t *, circ)
-)
-
-/*
- * General circuit events.
- *
- * Tracepoint use the circuit_t object.
- */
-
-/* Tracepoint emitted when a circuit is freed. */
-TRACEPOINT_EVENT(tor_circuit, free,
-  TP_ARGS(const circuit_t *, circ),
-  TP_FIELDS(
-    ctf_integer(uint32_t, circ_id,
-                (CIRCUIT_IS_ORIGIN(circ) ?
-                 TO_ORIGIN_CIRCUIT(circ)->global_identifier : 0))
-    ctf_enum(tor_circuit, purpose, int, purpose, circ->purpose)
-    ctf_enum(tor_circuit, state, int, state, circ->state)
-  )
-)
-
-/* Tracepoint emitted when a circuit is marked for close. */
-TRACEPOINT_EVENT(tor_circuit, mark_for_close,
-  TP_ARGS(const circuit_t *, circ),
-  TP_FIELDS(
-    ctf_integer(uint32_t, circ_id,
-                (CIRCUIT_IS_ORIGIN(circ) ?
-                 TO_ORIGIN_CIRCUIT(circ)->global_identifier : 0))
-    ctf_enum(tor_circuit, purpose, int, purpose, circ->purpose)
-    ctf_enum(tor_circuit, state, int, state, circ->state)
-    ctf_enum(tor_circuit, end_reason, int, close_reason,
-             circ->marked_for_close_reason)
-    ctf_enum(tor_circuit, end_reason, int, orig_close_reason,
-             circ->marked_for_close_orig_reason)
-  )
-)
-
-/* Tracepoint emitted when a circuit changes purpose. */
-TRACEPOINT_EVENT(tor_circuit, change_purpose,
-  TP_ARGS(const circuit_t *, circ, int, old_purpose, int, new_purpose),
-  TP_FIELDS(
-    ctf_integer(uint32_t, circ_id,
-                (CIRCUIT_IS_ORIGIN(circ) ?
-                 TO_ORIGIN_CIRCUIT(circ)->global_identifier : 0))
-    ctf_enum(tor_circuit, state, int, state, circ->state)
-    ctf_enum(tor_circuit, purpose, int, purpose, old_purpose)
-    ctf_enum(tor_circuit, purpose, int, new, new_purpose)
-  )
-)
-
-/* Tracepoint emitted when a circuit changes state. */
-TRACEPOINT_EVENT(tor_circuit, change_state,
-  TP_ARGS(const circuit_t *, circ, int, old_state, int, new_state),
-  TP_FIELDS(
-    ctf_integer(uint32_t, circ_id,
-                (CIRCUIT_IS_ORIGIN(circ) ?
-                 TO_ORIGIN_CIRCUIT(circ)->global_identifier : 0))
-    ctf_enum(tor_circuit, purpose, int, purpose, circ->purpose)
-    ctf_enum(tor_circuit, state, int, old, old_state)
-    ctf_enum(tor_circuit, state, int, new, new_state)
-  )
-)
-
-#endif /* TOR_TRACE_PROBES_CIRCUIT_H */
-
-/* Must be included after the probes declaration. */
-#include <lttng/tracepoint-event.h>
+#include "lttng_circuit.inc"
```

Comment:
We should include this by path, like we do for all our other include files: core/or/lttng_circuit.inc